corehq/motech/repeaters/migrations/0003_migrate_connectionsettings.py

```diff
@@ -0,0 +1,63 @@
+from django.db import migrations
+
+from couchdbkit import ResourceNotFound
+
+from corehq.motech.repeaters.models import Repeater
+
+DELETED_REPEATER_CLASSES = (
```

Comment:
Where does this list come from, and are you certain it is complete?

Reply:
I trawled git commit comments, and checked out what I found:
```bash
$ git log --grep=BETS180TreatmentRepeater
$ gco d64a302c35cc64f605f71d2ac828782108ed2455
```

But thank you for the prompting! I dug a little deeper ...
```bash
$ git log -S'(Repeater)'
$ git log -SSOAPRepeaterMixin
```
... and found more.